modules/creating-manifest-file-multiple-interfaces-br-ex.adoc

```diff
@@ -0,0 +1,187 @@
+
+.Procedure
+
+. Optional: Create an interface connection that your secondary interface, `br-ex1`, can use by completing the following steps. The steps use example configurations to show the use of two interfaces instead of a machine config manifest file to define a secondary bond interface for your cluster. 
++
+[IMPORTANT]
+====
+Do not use the Kubernetes NMState Operator to define or a `NodeNetworkConfigurationPolicy` (NNCP) manifest file to define the secondary interface.
+
+Also ensure that the secondary interface or sub-interfaces when defining a `bond` interface are not used by existing an `br-ex` OVN-kubernetes network deployment.
+====
++
+.. Create the following interface definition files. These files get added to a machine config manifest file so that host nodes can access the definition files. 
++
+.Example of the first interface definition file that is named `eno1.config`
+[source,ini]
+----
+[connection]
+id=eno1
+type=ethernet
+interface-name=eno1
+master=bond0
+slave-type=bond
+autoconnect=true
+autoconnect-priority=20
+----
++
+.Example of the second interface definition file that is named `eno2.config`
+[source,ini]
+----
+[connection]
+id=eno2
+type=ethernet
+interface-name=eno2
+master=bond0
+slave-type=bond
+autoconnect=true
+autoconnect-priority=20
+----
++
+.Example of the second bond interface definition file that is named `bond1.config`
+[source,ini]
+----
+[connection]
+id=bond1
+type=bond
+interface-name=bond1
+autoconnect=true
+connection.autoconnect-slaves=1
+autoconnect-priority=20
+
+[bond]
+mode=802.3ad
+miimon=100
+xmit_hash_policy="layer3+4"
+
+[ipv4]
+method=auto
+----
++
+.. Convert the definition files to Base64 encoded strings by running the following command:
++
+[source,terminal]
+----
+$ base64 <directory_path>/en01.config
+----
++
+[source,terminal]
+----
+$ base64 <directory_path>/eno2.config
+----
++
+[source,terminal]
+----
+$ base64 <directory_path>/bond1.config
+----
+
+. Prepare the environment variables. Replace `<machine_role>` with the node role, such as `worker`, and replace `<interface_name>` with the name of your additional `br-ex` bridge name.
++
+[source,terminal]
+----
+$ export ROLE=<machine_role>
+----
++
+[source,terminal]
+----
+$ export INTERFACE=<interface_name>
+----
+
+. Create a machine config manifest file for configuring the network plugin by entering the following command in your terminal:
++
+[source,terminal]
+----
+$ oc create -f <machine_config_file_name>
+----
+
+. Create an Open vSwitch (OVS) bridge, `br-ex1`, on nodes by using the OVN-Kubernetes network plugin to create an `extra_bridge` file`. Ensure that you save the file in the `/etc/ovnk/extra_bridge` path of the host. The file must state the interface name that supports the additional bridge and not the default interface that supports `br-ex`, which holds the primary IP address of the node. 
```

Comment:
We need to move this step to around line 159 --> before 
"Bind `br-ex1` to the secondary interface by configuring the `/var/lib/ovnk/iface_default_hint` file." block.

Reply:
:heavy_check_mark: